### DISCUSSION TITLE
Final Thinking Cap💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#27] [Assignments]
### Due Thursday 6/8

Respond to the following questions: 
> ## What are your thoughts about the future of web development for civil society? 
> ## Which visiting voice left the most impact on you?
> ## What personal connections to people (if any) would you like to maintain from this class going forward?

## Submission
Share your thoughts in at least **4-5 sentences** before Thursday, 6/8 and respond to **one** other student.

**Comment by cassiuscarandang** [comment]
The future of web development for civil society can be difficult to tell from here. Society can hold a tremendous amount of potential for positive change and impact. But considering the methods in ethical views, validity, open and closed sources, design/data justice, and many other factors, it is crucial for developers to acknowledge these components as they shape the next generation of web development. Rapi Castillo has made a perfect example of staying optimistic towards positive change and impact. As he is a community organizer and tech worker, he uses open-sources to utilize the community throughout different grassroots organizations he is involved in. By visualizing and creating a story, we are in this space of technology in which he vows to fully advocate for equity. From receiving unclaimed benefits to providing services that can benefit insecure individuals, his nonprofit organization helps reduce unethical power and provides fair-mindedness for all. For me personally, I do not have any personal connections to people to maintain but more so connections I would like to maintain from this class (lectures and lessons I learned) going forward is to be ethically aware of the actions I utilize or create in web development. Because that will either make civil society 1% better or worse.

**Reply by paige-lee** [reply]
I agree with you that Rapi Castillo’s presentation was very positive and optimistic. I think in general, it’s important to have a positive attitude when working towards social change. Social injustices tend to be distressing topics, and further, we may not notice our impact or progress immediately.

**Reply by jamesfalkenroth** [reply]
I also enjoyed Rapi Castillo's presentation, as he showed how to empower grassroots organizations with open source mapping. It's good to think about our actions as making civil society 1% better or worse based on the work we do. The changes may be so gradual that we don't notice them on a day to day basis, but over time, the changes will compound. It will be beneficial to keep the lessons you learned in this class in mind going forward.

**Comment by paige-lee** [comment]
I think web development, when used responsibly with a focus on ethics and social justice, can greatly support civil society. When collecting and handling data as well as building websites and applications for members of civil society, developers should let community voices guide decision making. With the recent, rapid advancement of AI, it is our responsibility to use the power of technology to benefit humanity rather than allow the perpetuation of existing social justice issues through biases and discrimination for instance. The visiting voice who left the most impact on me was Rapi Castillo. He presented many impactful examples of his past work, which serve as relevant, real world inspiration for the social justice projects we’re developing in this class. For instance, the map he created for Bernie Sanders’ 2016 presidential campaign had category-specific markers for campaign events, which are features that we’ve learned how to make in class. It also allowed users to filter sites by proximity to an input zip code, which I would have been interested in learning how to implement in class if we had more time. I also liked his project that mapped all scheduled buses from each city to the 2017 Women’s March in D.C. in response to Trump’s inauguration because as a statistics major, I am very interested in dealing with and querying big data. There aren’t any particular personal connections I’d like to maintain from this class. Rather, I’d like to take with me the principle of respecting and prioritizing community voices in web development. This is a key strategy that is necessary for addressing social justice issues, and it is something I really struggled to grasp and implement in my group project this quarter.

**Reply by danigold1020** [reply]
I agree with your point about web developers needing to collaborate with community members to help guide their decision-making process as this will help ensure that the future of web development plays an empowering role in society. I also like that you mentioned that it has been a challenging experience to prioritize community voices and focus on the most essential needs of a community throughout our group projects this quarter. It definitely is easy to get caught up in wanting to propose a solution or feeling that we understand a community without talking to its members, so this is why it is important for our projects to provide a space for our communities of focus to share their stories/perspectives so that we learn from them.

**Comment by danigold1020** [comment]
I'm excited about the potential for web development to serve as an accessible, educational tool for our society and thus believe that coding skills and web development knowledge should be taught early on in schools. Many of us have lived our lives viewing different websites without thinking about how they are created, thus making us feel like we may not be "smart enough" to understand how technology works or are not "high enough" in society to design technology that can be used by the public. However, learning how to design a website early can help prevent this feeling in future generations and enhance their confidence by making them not only feel more connected to technology, but also more involved in the influence of technology on society. It is important to note that the automation aspect of artificial intelligence may reduce the need for manual web development, thus curbing the progress that can be made by teaching children how to code/design a website early. 

I really enjoyed listening to Nancy and Farrah's presentation about their work with CAUSE. I like that they emphasized that we should care about communities (e.g. districts) that are not our own, because those communities may face events/challenges that may require our assistance and their effects can trickle down and affect us in ways we may not even know. I appreciated that they demonstrated the ways that CAUSE has promoted its organization/mission across various platforms and how their use of maps has been crucial in helping the public understand their work. I also liked that Farrah mentioned that she negotiated her job with Nancy as she continued her role in CAUSE. It was very empowering to hear as a graduate entering the job market that many employers don't know what they need, so job candidates can highlight their skills and passions to address employers' current and future needs.

I would like to maintain a connection to our class community as a learning resource (about web development, technology, artificial intelligence, etc.) moving forward.

**Reply by zacharypakin** [reply]
I completely agree that coding/web dev skills should most definitely be taught in schools at a young age and how for most of our lives we take for granted the websites we use everyday. Coming into college and being introduced to a completely new area of study is very daunting, therefore introducing the topics at a younger age would reduce the information overload. I also agree that Nancy and Farrah's presentation was very engaging and eye opening!

**Comment by zacharypakin** [comment]
I believe that the future of web development for civil society will be bright, as long as advocacy towards accessibility, design justice, and data justice continue. Web development for civil society is, in the scope of the internet's lifetime, rather young, providing immense potential for growth over the coming years. As more individuals learn about the abilities (and limitations) of web technologies to amplify voices and raise awareness, I believe that the usage and usefulness will flourish. The visiting voice that left the most impact on me was Mariah's, as I feel as she set the stage for the course very well. Learning about how all representations of knowledge can be considered maps and how the maps we are traditionally accustomed to are biased and can be harmful made me think more critically about the decisions made on web projects every day. I especially found it interesting when we were told about auditory maps, as this was the most surprising to me. I had never heard of someone refer to a spoken geographic description as a map, which was very thought provoking. In terms of connections I would like to maintain, I would love to remain in contact with my group mates. Additionally, as I am moving to New York in Fall, I may reach out to Rapi Castillo after I am settled in.

**Reply by shintuzd** [reply]
I like how you mentioned that Mariah's visiting voice set the stage for the rest of the course, and I do agree with you. Her presentation highlighted the way maps can be read and interpreted, and how it comes in different forms. It depends on the storyteller to decide on the medium, not the designer, which is where things like design justice come into play. This is why the principles you mentioned - accessiblity, design justice, and data justice - are key to a bright civil technoglical society. Otherwise, we end up back at the status quo of power inequities.

**Comment by shintuzd** [comment]
My thoughts about the future of web development for civil society are mixed. On the one hand, I am certain that the principles we learned in this class, such as design and data justice, will make for a bright future for civil technology. However, we rarely see these principles in day-to-day web development, therefore I can’t imagine much change being made immediately. As we learn these skills, it is important we share and utilizes these techniques in our professional work moving forward to ensure the future of web development for civil society is as participatory and empowering as possible. 

One of the most impactful visiting voices has been Yoh Kawano. Not only did his presentation highlight the importance of building trust with communities to effectively share their stories with the world, but he also showed how the skills we learned in this class are applicable across the world. Although I did not get to meet many peers in this class, my hope is to maintain a personal connection with our very own professor. He has been helpful in providing guidance with growing our technical skills, and how to apply it to civil society. To continue this line of work, I am sure he will be a great source of mentorship down the road.

**Reply by cassiuscarandang** [reply]
Hi, I agree with the idea you brought it with rarely seeing the principles of day-to-day web development being utilized as a community and empowering effort. To me, sharing these techniques and strategies to civil society can be helpful in ways that there can be potential growth for so many users and web developers. But I see why you feel mixed due to how people will use these tools (for better or for worse). I believe that by having professional teachers of web development are vital towards providing civil society the tools they need to achieve their endeavors and work towards an innovation that can better the technological world for all.

**Reply by JonathanGLopez** [reply]
Hey Mohammed, while we don't necessarily see design justice implemented every day, we do see the use of UX/UI design being implemented every day to design the interface of software we interact with on a daily. However, UX/UI design is usually used to make interfaces addicting whether the creators are aware of it or not, and does not include participatory research or design justice. However, in the coming years, we can see the field of UX/UI design become an avenue to implement data justice. As seen in the Netflix documentary the social dilemma, many designers and engineers at top companies are aware that is current software is addicting and we need to find ways to make the design less addicting. This means that we are seeing the conversation start around design justice concepts.

**Comment by jamesfalkenroth** [comment]
I am somewhat optimistic about the future of web development for civil society, as we have seen many people who do good work in the field. I have enjoyed learning about how to consult the community, use open source tools, and keep ethics in mind when designing. Unfortunately, this type of work is not generally incentivized financially. When most people think about technology, they probably imagine large corporations like Google and Amazon. These companies usually pay well and focus on profit instead of benefiting civil society. I think it's good to have more classes like this one, that will teach people to keep civil society in mind throughout their careers. Hopefully as people spread awareness about the importance of community involvement in technology, the trend shifts towards more consideration of civil society within tech companies. 

To me, the most impactful visiting voices were Nancy Yap and Farrah Su and their work with CAUSE. It was inspiring to see how their activism work, mapping, and outreach efforts led to real policy impacts. They were able to help prevent a congressional redistricting in San Gabriel Valley that would have split the AAPI community and their electoral power. They were also able to help establish new voting centers in Chinatown and Koreatown, based on community input about where they were needed most. Their project with CAUSE showed me the power of organizations dedicated to community justice and activism. These organizations can benefit from people with the technical skills to support their goals. I hope to work with a company or organization where my skills can do some good. 

After this class, I hope to maintain connections with Professor Kochaphum, with my group members, and with others in the class who I've met during my time at UCLA.

**Reply by czhangy** [reply]
I definitely agree that courses that focus on the societal impact of technology should be more common, especially in higher education. I personally took this course because there was nothing like it in the Computer Science department, and I felt that a deeper dive into the topic would benefit me in my career moving forward. Your point about tech companies being profit-driven definitely rings true from my experience and, like you, I hope that this mentality can change in favor of civil society moving forward.

**Comment by czhangy** [comment]
I think that the future of web development is very bright. More and more people are learning how to make websites, which means there is an increasing number of people with access to one of the most flexible mediums of communication/expression in the world. However, as we've learned in this class, while the people technical knowledge has been increasing, it's necessary for the ethical knowledge to follow suit. Concepts like design/data justice and open-source software are critical components of this development, and I believe that they will be recognized as such as the technology continues to become more accessible.

To me, the most impactful visiting voice was Joyce, since she really brought the issues with Asian Americans in elections and academia to my attention. Although I had never really given these domains much thought, the issues she faced really resonated with me and have definitely stayed on my radar since. I would love to stay connected to my group members moving forward.

**Reply by kmtsujimoto** [reply]
I completely agree that there's hope in the increasing accessibility of web development to provide a medium of communication and expression to more communities, especially those who are underserved and potentially historically underrepresented in computer science development and academia. From one of our first lectures, I remember seeing a slide full of mostly older White men as we learned about the history of GIS and web development. Accessibility will open new doors to a wider variety of demographics. In turn, that will hopefully translate through the application of design and data justice principles to productive social change.

**Comment by kmtsujimoto** [comment]
I think that while web development has a lot of potential for civil society, there is a still a lot of work to be done to bridge the gap between communities and technology. As a society, we tend to drift towards technological determinism, believing that the newest technology (i.e. ChatGPT) is inherently productive and positive. However, we don't always consider the full repercussions of design and development on communities. With increasing accessibility to coding proficiency and technology, design justice becomes a relatively attainable goal, but a greater step needs to be taken to implement data justice and data justice teachings into coding education. Only through classes like this one have I been able to truly understand the impacts of the products we create on and for communities. We need to be more intentional and deliberately aware in our development in order to avoid reinforcing existing social inequalities in the communities we are trying to serve.

I really enjoyed the visiting voice lecture from Yoh Kawano. He talked about a project on Vietnam sidewalks which helped me to understand the importance of unconventional and out-of-the-box mapping in order to visualize those who occupy spaces that are not seen on typical maps. Moreover, his work with the Fukushima nuclear power plant/earthquake/tsunami disaster showed me the impact of giving voice to forgotten narratives and demonstrated how the skills we are learning in this class can be productively applied to give voice to target communities while empowering but not extracting.

I would like to maintain connections from this class to the wide diversity of majors and backgrounds within our classroom community. I appreciated the diversity of voices and skill sets that each brought something unique to the table. I would also especially like to stay connected with my group and Professor Kochaphum whose guidance and mentorship really shaped the way I understood justice in GIS and web development for social change.

**Reply by emai14** [reply]
I also really enjoyed Yoh's lecture. I think the projects he talked about really opened my mind about how to visualize and utilize maps for research in different ways! I'm glad you brought that up.

**Comment by emai14** [comment]
I feel that people are becoming more aware of the impacts of web technologies and are now working towards dealing with those injustices. I think that because of this, I am somewhat optimistic about the future of web development for civil society. The fact that design justice, data justice, and these types of content are taught at a public university is promising to me. Of course, there is always more to do, but I'm glad these things are being addressed. I'm still scared at what the growth of these technologies brings to society, as I feel that laws and activism cannot necessarily keep pace with the speed at which capitalism causes damage. Still, I'm glad that work is being done to alleviate the digital divide, empower communities, and other issues caused by web technologies.

I feel that I was the most impacted by Mariah. I think she really set the tone for the whole class, and put me in the right mindset to start thinking about mapping for social change. Moving forward, I hope to stay in contact with her, especially because I want to eventually do the type of work she does with advocacy against the carceral system.

**Reply by catherinehhu** [reply]
I agree that it's nice to see these topics being addressed, especially in a class like ours, but it is scary to think about whether or not education/conversation can keep up with the ever-changing ways people use technology. I really resonate with what you said about Mariah setting the tone for this class because I think her presentation was really the one that got me excited for this course and for thinking about how maps can be used to create change.

**Comment by catherinehhu** [comment]
It’s heartening to see how many resources are out there in the world for people who are learning about programming and web development. With that said, however, the future of web development in the context of civil society is something we need to continue to have active conversations about, especially regarding what it means to build something with and for a community. It’ll be easier and more accessible than ever to make a website, but learning how to be conscious of the content and narrative portrayed on that site is still something we need to emphasize and highlight. Personally, Mariah Tso was the most impactful visiting voice because her perspective felt so different from what I’m used to hearing about maps. As someone pursuing a minor in GIS, I hope to carry her insights with me in understanding the power and impact of maps. As for connections with this course, I’d love to stay in touch with our class (ie. through our Discord) as we all continue to learn and explore.

**Reply by chrysanthemma** [reply]
Wow, I didn't know you were minoring in GIS, that's really cool! (I didn't even know what GIS was until this class haha.) As you are a GIS minor pursuer, I wonder how much they talk about data / design justice in the other required courses? I've learned about UI/UX here and there before, but no one has really ever mentioned either concept really. I have heard about empathizing with the user a lot though. Anyways, I agree that we should be careful as to what kind of content we are creating and who it impacts, rather than simply creating just to create, which I feel like can be the rhythm for certain advancing technologies.

**Comment by chrysanthemma** [comment]
The future of web development for civil society is a bit scary. From the advancing technology of AI to the large amount of data collection pulled from users, it's more common to come across unethical web development that serves corporations rather than the community. On the bright side, there seems to be a growing accessibility to learning coding and web development, and we've seen in this class many different projects used to uplift specific communities. The visiting voices that left the most impact on me were Nancy Yap and Farrah Su. It was great to see a real life application of mapping used in the local community and the process behind it. I'd like to continue to the connections I've made with the class, and hope to continue learning about how to implement data and design justice.

**Reply by maizaha** [reply]
Definitely agree that the future of web development is scary! Unethical data collection and usage is such a big problem, and there are scarily few laws and restrictions in place preventing unethical use. Hearing all the guest speakers talk about how they're considering the issues with technology while also leveraging it for social justice is reassuring though (and also your points about growing accessibility + consideration of data/design justice), and I also think that Nancy and Farrah's real-life application was really inspiring.

**Comment by maizaha** [comment]
The future of web development for civil society generally feels a bit worrisome - the mainstream tends to take a heavily technological determinist approach to using technology/web development to solve problems, and data is often taken at face-value as unbiased and all-powerful. Media hype surrounding new technological developments (like AI, ChatGPT, etc) often shrouds critical evaluation and discussions of ethics. However, as this class has shown me, there _are_ individuals and communities working towards leveraging tech/web development for social justice while at the same time remaining critical and cautious. I really hope that discussions of justice and ethics will steer technology in the right direction (or act as a strong counterforce to hold tech accountable), at least in certain circles that are dedicated to highlighting community voices and tackling systemic issues from their roots. 

That being said, Mariah Tso was the most impactful speaker for me personally - I loved the way she brought her own insight as an Indigenous geographer into the (extremely important, creative, and impactful!) work that she does. The way that she spoke about maps and knowledge/power systems really altered the way I think about technology, mapping, and knowledge - I feel a lot more hopeful knowing that it's possible to leverage these tools to flip Western colonial power dynamics and tell the stories that are too often left out and repressed. I hope to keep in touch with Mariah in the future (I'm really interested in the Million Dollar Hoods project!), as well as both our professor and fellow classmates as we've learned so much on our journey together this quarter.

**Reply by joanneluong** [reply]
I agree that people don't often think twice about data that's handed to them, which can lead to scary outcomes when the data may be presented poorly. I also like what you brought up about the hype behind new technology and how it brings up the ethics of these tools. I think that humans naturally take a interest to whatever is the newest, shiniest toy sitting in front of them, and right now that thing is AI. There is a lot of discussion about the ethics of it, but there are many who use it regardless of the impacts that it has on society.

**Comment by joanneluong** [comment]
The future of web development for civil society is definitely uncertain and is bound to change in these comings years, especially with the new rise of AI. I think that where it stands right now, there's always room for growth and reflection, especially considering the various readings we've been introduced to throughout the quarter. It will definitely be difficult to learn how to navigate ethical web development in the new technological landscape, but I think as long as there are still people who care about the issues at their core and want to do what is best for their communities, things will still be ok. The visiting voice that left the most impact on me was Yoh, because I was really interested in the work that he did and how he applied his experiences and skills to the community. I would like to maintain a connection with the professor as well as my peers and see how their projects live on past this class!

**Reply by SageLuong** [reply]
I love your optimism regarding the future of web development for civil society! I also believe that as long as we have considerate, humanistic-minded people, creators will be able to uplift and collaborate with communities in a way that disperses knowledge and skills. I also enjoyed Mr. Yoh's talk on how we went from a more technical project to one that used his technical knowledge in a way that explored community stories devastated by nuclear aftermath. It's reassuring to know that not all skilled professionals like Yoh immediately know how to make a community-based project that is respectful to the community they represent. That way, I'm not so hard on myself as I try to think about the harder questions in creating anything whether that's UI, websites, or graphics.

**Reply by trufflelover** [reply]
I agree with you that the future of web development is definitelly uncertain and for me, I am excited but also scared if the AI will be manipulated by the more dominant group of society. I agree with you that Yoh's work was definitely very interesting! I also aspire to be able to work in the United States but also at another country doing projects based on the new country. I think that experience would truly expand my cultural perspective!

**Comment by SageLuong** [comment]
If creator focus less on simply creating web applications for the sake of innovation, I feel like the future of web development for civil society can bring many benefits to various communities. There is always so much emphasis on improving website functionality, UI, and efficiency, but not as much WHY we want to improve these aspects. I feel that adding more mindfulness to our creations as a form of empowerment and awareness (rather than solutions) will be helpful in adding meaningful improvements to society that actually affect disadvantaged people and not just flex the technical skills of professionals. My favorite vising voice was CAUSE because it reminded me of how much change a small group of concerned citizens can bring in such a short time period with limited resources. The speakers demonstrated that even though knowledge of advanced technology is helpful in streamlining the process, it is not necessary to connect with communities and hear their stories and concerns. We are all on our process of applying technology to humanistic problems in a way that explores problems rather than solves them. Besides the technical skills I learned in class, I met some nice people that I'd like to keep in contact with through online means! Everyone is so passionate about what they do, and I'd love to know what people make in the future, even if it's not related to mapping or civil society.

**Reply by alanamalika** [reply]
Hi Sage! I completely agree with what you said about UI. There is definitely a lot of conversation about optimizing product development for the “user” but I think integrating ethics grounds developers to think about the personhood of a “user.”

**Reply by DylanJWan** [reply]
I agree completely with your take on improving UI. I think only focussing on improving efficiency of UI has the effect of "datafying" users which does the opposite of uplifting them.

**Comment by alanamalika** [comment]
Personally, I think the future of web development is currently at a crossroads. I felt increasingly optimistic when I read about design justice, data justice, and all of the other ethical implications in web development. Moreover, it was refreshing to hear from all of the visiting voices to see and hear about how professionals are committed to practice this values. I think the boom of AI will force the mainstream discourse in the tech industry to critically think about the best practice of web development from an ethical perspective. Amid a precarious economy and tech layoffs, a lot of high-skill workers are contemplating the sustainability of corporate tech and I hope this could lead to a shift into civil pursuits within the industry. Yoh Kawano’s lecture resonated with me the most, particularly because I recall looking through his project, HyperCities, long before I took this course. It was interesting for me to hear about his work in Fukushima. From this class, I hope to keep in touch with Professor Albert and a few of my peers. Moreover, I hope that I can continue to peruse the class website because there are so many resources I would like to revisit in the future!

**Reply by trufflelover** [reply]
I agree with you that Yoh's lecture as very inspiring. I also think his projects are interesting and what he is doing in Japan. I think it is also very cool how he has worked both in the States and Japan and how he gets to experience the different environments. However, I personally thought Lina's lecture was very inspiring as well and I really enjoyed her pirates themed presentation.

**Reply by stellakangj** [reply]
It's great to hear that Yoh Kawano's visiting voice lecture made a significant impact on you. His work on unconventional mapping and giving voice to marginalized communities demonstrates the power of using web development skills to empower and uplift those whose narratives are often overlooked. This serves as a reminder that technology can be a tool for positive change when applied with empathy and consideration for the communities it serves.

**Comment by trufflelover** [comment]
My thoughts regarding the future of web development of civil society is that it will heavily be depended on the ethics and morals of AI like chatGPT. As more and more people integrate using AI in their life, including web development, it is cruicial that the AI is not shaped by the dominant group of society and that they don't project results that implicitly favor the dominant group over the other. The visiting voice that had the most impact was Nina Kin. Personally she had the most impact because it was relieving and inspiring to learn how she got where she was today. I have been feeling lost not knowing what I am doing, she made me feel better and safe. I would truly like to keep in touch with Professor Albert and my group mates. I am so glad I took this class because I got to meet amazing people and learn so much.

**Reply by cassidytu** [reply]
I definitely agree that ethics and morals are crucial to the future of web development! Emphasizing, sharing, and learning about how these factors in web development impact civil society is one way of moving towards the right direction. Nina Kin was a lovely speaker! It's difficult to see what the future will look like right now, but I'm sure that you will do and achieve great things!

**Reply by qatle** [reply]
I agree that the future of AI will be heavily depended on ethics and morals of the data and the people take control behind it. It's also tricky, even my Professors who teach CS here predicted that their jobs will be gone in 10+ years, but I'm positive that (and most professionals also agree), as long as you're updated with both technology and current society, you would be able to survive and be fine. Before this class, I never thought of applying tech to anything else besides ... tech, but this class has taught me how I can contribute my skills to society, how important my skills could be if I apply them to the right place and there are many places that could use my help due to the lack of technology. I'm thankful that I chose this class over another CS class, because this has taught me to open my mind more and  not just tie myself to "prestigious" titles but rather things that I believe are meaningful.

**Comment by cassidytu** [comment]
The future of web development has a lot of flexibility and room for growth, ultimately leading to consequences that could be positive or negative to civil society. Understanding the different aspects of web development, from design and data justice to the mindset of technological determinism, are ways to fully grasp how have meaningful digital spaces rather than a disconnected "solution" to an issue. One of the most impactful visiting voices was CAUSE, with Nancy and Farah. As a smaller organization that facilitates ways to raise awareness and prevalence of Asian American voices in an area that I call home, it was inspiring to see the level of impact they could create with skills that we are being taught in this very class! I hope to remain in contact with them, as well as Professor Albert and fellow peers who have shared interesting insights and learning experiences with me.

**Reply by ariscorpus** [reply]
Web development definitely has a lot of flexibility and room for growth, but I feel it is in a solid spot at the moment. I also really liked Nancy and Farah's visiting voices as they show how Asian American voices are advocating for their communities and areas that they call home. Being able to see real-life applications within concurrent issues within our community is something I found to be so interesting and I'm happy to see that you felt the same way!

**Comment by ariscorpus** [comment]
I feel as if web development is in an excellent state in which developers are beginning to recognize how they can empower communities and civil society as a whole. Throughout this class, we learned about different aspects of data implication such as data justice and the ethics around data usage, which stems from continuous development into the community aspect of web development. This to me, places web development in a very good spot for the future and makes me optimistic about how we can implement technology within our society. I really enjoyed Nancy Yap and Farrah Su's presentation as it shows me how we can apply what we learned in class to real life. I personally do not have many personal connections in this class but what I was able to learn in this class overall is something I hope to maintain in my future endeavors.

**Reply by angelicaalacantar** [reply]
I agree that web development in its current states allows for infinite ways for developers to be able to empower communities. There just needs to be more of a push or demand for developers to take on such endeavors for civil society to reap the benefits more and more.

**Comment by angelicaalacantar** [comment]
The future of web development for civil society seems to be evolving gradually in the right direction of empowering communities, as we saw with the various community-based projects that visiting voices worked on. However, there is still a lot more emphasis on topics like design and data justice in order for there to be more effort and innovation dedicated to civil society rather than private agendas. The very first visiting voice, Mariah Tso, in particular was a visiting voice that stood out to me in how she set the stage for this class and immediately opened my eyes to mapping in a a new and more critical way. Her lecture in particular was very important when approaching the final project for this class as well as can be applied to real life beyond the class. Critical cartography is a very important concept that should be more known in the general public. It would be nice to be able to maintain connections with my peers in this class if possible and get to see what projects they may create in the future!

**Reply by vickydelk** [reply]
Hi! I also really loved hearing from Mariah Tso, I think that after listening to them I was made aware of the power mapping has towards helping civil society. I also agree that the future of web development is slowly going in the right direction. We have already made many strides towards user and community protection and with the rise of open source, I am hopeful that we will continue to progress.

**Comment by vickydelk** [comment]
As web development and the increasing importance of online presence increase, we will slowly move in the right direction toward empowering civil society. I think that already the importance of privacy and data protection has been highlighted and has started to become a necessity for tech companies to follow. Also, I think that open source will become more regular which should aim to create a more transparent and collaborative development environment. The visiting voice that had the most impact on me was Nancy Yap and Farrah Su's since I felt personally connected to their mission and felt like they showed concrete examples of ways to empower a community. Going forward I hope to remain connected to my group mates asI really enjoyed collaborating with them and felt like our priorities and concerns for society aligned.

**Comment by JonathanGLopez** [comment]
The readings surrounding design justice and data justice make me optimistic in believing that we will see a growing body of volunteers, nonprofits, and consulting firms shifting who builds and designs technology and websites, and who gets to access them and use them for civic society. Mariah Tso was the visiting voice that had the most significant impact on me. Her short presentation was presented in a clear and concise way and makes one evaluate the power dynamics that determine data visualizations. Her presentation also discussed the data silences we may see due to the power dynamics embedded in our capitalist country and the academy. I would like to stay connected with you Professor Kochaphum. Even though I struggled with your labs a bit, that is a reflection more of my quarter and lack of coding skills. But your class and lectures are very unique and my class and UCLA were privileged to have a professor who critically asses the role technology plays in civic society and the roles it plays in fueling inequality and systemic issues. I would like to stay connected with you to continue to learn about web development and GIS while ensuring that it is focused on civic society and people with the purpose of social change.

**Comment by qatle** [comment]
Web dev for civil society will definitely be huge, whether it's a good or bad thing will heavily rely on the developers and the people planning behind them. It's essential to be aware and be responsible with what you are reading and showing to connecting people.  For me, all speakers left different impression and there's at least one lesson to learn from each of them. It may sound cliché but I think all of them left me equal impact that I feel like without any of them it would be a puzzle with missing pieces.  For connection in this class, I hope I can still connect with my group, knowing more about people in this class, and chill with Professor and talk about anime, games or whatever aspect in life that we can agree on. I also want to keep up to date with your future journey and know what you do to help society :)

**Comment by DylanJWan** [comment]
I think the future of web development for civil society is bright. As time passes, more people are becoming aware of and prioritizing the implementation of data justice and design justice into their practice. I think the visiting voices that left the biggest impact on me were Nancy Yap and Farrah Su. I was inspired by their very immediate response to the congressional redistricting proposal that would directly impact the Asian community. From this class, I would like to maintain connections with Professor Kochaphum.

**Comment by stellakangj** [comment]
The future of web development for civil society holds immense promise, but bridging the gap between communities and technology requires concerted efforts. This class has shown me the importance of integrating data justice and design justice into web development to consider broader consequences. Yoh Kawano's lecture, particularly his projects on Vietnam sidewalks and Fukushima disaster, deeply impacted me by highlighting the power of unconventional mapping and amplifying forgotten narratives. It demonstrated how our skills can empower target communities. Through this class, I gained a deeper understanding of community impact and the need for intentional development. Overall, I particularly value the connections I've formed with classmates, group members, and Professor Kochaphum, and I hope to maintain these personal connections beyond the class.

**Reply by mandymwlai** [reply]
I was also very captivated by Yoh Kawano's work on Vietnam sidewalks and Fukushima disaster. Before taking this class, I was not aware of the existence of GIS technology and the applications of it. His work on the Fukushima disaster shows me how spatial data can be used in a meaning manner, to explore the aftermath of a natural disaster and to give voices to marginalised communities who're not covered by mainstream media. It was also striking to see how the 311 disaster played out after 12 years.

**Comment by mandymwlai** [comment]
I have mixed feelings about the future of web development for civil society. On one hand, I am deeply pessimistic due to the profit-driven nature of the current tech industry. Tech giants seem to disregard the well-being of their users, exemplified by the popularity of apps like TikTok that foster smartphone addiction for hours on end. On the other hand, I find optimism in the younger generation, particularly in Gen Z, who possess a more conscientious mindset. Having grown up with technology, they are very aware of its impact on human lives and I believe they will mature into responsible web developers.

Out of all the visiting voices, Mariah Tso’s sharing on Maps & Colonial Violence was the most impactful for me (Yoh Kawano was a close second). Her sharing opened my eyes to the many different types of medium that can be a map, be it bee hives, green-book or auditory maps. I would hope to stay connected to my group members going forward. It has been a great experience learning and collaborating with them in the past quarter.

**Reply by amandatlxu** [reply]
I agree that it can sometimes be very discouraging when considering the current climate of the tech industry. The “tech giants” often seem unstoppable, and tend to utilize technology and web development in not so ethical ways. However, I think concepts such as design justice and data justice are growing in popularity, so I am hopeful that the future of web development is more beneficial to civil society than it is currently. Concepts such as ethicality and accessibility are also an increasingly discussed topic that may aid this as well.

**Comment by amandatlxu** [comment]
I look forward to seeing further improvements in regards to web development for civil society. I think in our current social climate, web development is not typically used in a way that helps civil society, and rather is primarily utilized for selfish means such as for profit. However, with concepts such as design justice and data justice in mind, web development can be utilized in a way that helps civil society and social issues overall. The visiting voice that stood out to me the most was Yoh Kawano. I really enjoyed hearing about the concept of urban data science, and building human narratives through code. A part of his presentation that really stuck with me was when he mentioned “mapping outside the box” and thinking about what is behind your map and the purpose behind these maps. Moving forward, I hope to potentially utilize the contact information from the many visiting voices we heard from this quarter! I also would like to maintain connections with Professor Kochaphum, my groupmates, and my other peers in AsiaAm 191A this quarter.
